FormatterKit/TTTTimeIntervalFormatter.m

```diff
@@ -120,12 +120,12 @@ - (id)init {
     self.locale = [NSLocale currentLocale];
     self.calendar = [NSCalendar currentCalendar];
 
-    self.pastDeicticExpression = NSLocalizedStringFromTable(@"ago", @"FormatterKit", @"Past Deictic Expression");
```

Comment:
Might make sense to just grab a reference to the bundle where this is used repeatedly so you don't have to keep repeating the whole bundle for class bit over and over and over again